drizzlepac/adrizzle.py

```diff
@@ -309,13 +309,13 @@ def run(configObj, wcsmap=None):
     out_sci_handle.close()
 
     if not util.is_blank(configObj['outweight']):
-        out_wht_handle, outwhtext = create_output(configObj['outweight'])
+        out_wht_handle, outwhtext = create_output(configObj['outweight'] outwht)
```

Comment:
For the want of a comma ...
Comma is missing between **configObj['outweight']** and **outwht**.

Reply:
good catch!